base/int.jl

```diff
@@ -57,18 +57,28 @@ typemin(typeof(x))`, `abs(x) == x`, not `-x` as might be expected.
 abs(x::Signed) = flipsign(x,x)
 
 """
+    Base.checked_abs(x)
+
+The absolute value of `x`, with overflow error trapping where applicable.
+For types for which no overflow can happen during `abs`, this is equivalent
+to `abs(x)`.
+"""
+function checked_abs end
```

Comment:
Yes, LGTM.

RE the big int test thing: it's not the only bigint in test/int.jl and they'll be easy to find when/if they're every removed from base. So I think we can merge as is.